storm-server/src/main/java/org/apache/storm/scheduler/resource/strategies/scheduling/DefaultResourceAwareStrategy.java

```diff
@@ -79,7 +79,8 @@ public SchedulingResult schedule(Cluster cluster, TopologyDetails td) {
         }
 
         executorsNotScheduled.removeAll(scheduledTasks);
-        LOG.debug("/* Scheduling left over task (most likely sys tasks) */");
+        LOG.debug("Scheduling left over tasks {} (most likely sys tasks) from topology {}",
```

Comment:
any idea why this is happening?

Reply:
First round of ordering executors only includes topology components. We need this round to schedule left over system components.